app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -33,7 +33,11 @@ def pretty_print(classes)
       end
     end
     hash.keys.sort.map do |key|
-      link_to key,{}, {:remote => true, :rel => "popover", :data => {"content" => hash[key].sort.join('<br>').html_safe, "original-title" => key}}
+      link_to key,{}, { :remote => true, :rel => "popover", :data => { "content" => hash[key].sort.join('<br>').html_safe,
+                                                                       "original-title" => "#{key} has <span class='label label-info'>#{hash[key].size}</span> classes",
+                                                                       "trigger" => "manual", "toggle" => "popover"
+                                                            }
```

Comment:
Bracket indentation doesn't match. I would suggest also making to outer hash line-seperated.